src/locale/properties/shared/shared.en.properties

```diff
@@ -95,7 +95,7 @@ share.sortOrder=Order
 shared.activities=Activities
 shared.addToList=Add to list
 shared.organization=Organization
-shared.organizationError=Please enter a organization
+shared.organizationError=Please enter an organization
```

Comment:
Dani, please create a new translations string for this one, deleting the old one. 

This is to avoid confusion during the re-translation process

Reply:
done!